package.json

```diff
@@ -25,61 +29,62 @@
     "node": ">=4"
   },
   "dependencies": {
-    "btoa": "^1.1.2",
-    "codemirror": "^5.13.2",
     "express": "^4.13.3",
-    "extract-text-webpack-plugin": "^1.0.1",
-    "filesize": "^3.3.0",
-    "kinto-client": "^0.8.3",
```

Comment:
Just to confirm, we're saying that the only true "dependencies" of this project are the ones we use to serve it, and everything that is part of the code that we actually use when we run it goes into "devDependencies"? Seems strange, but OK.